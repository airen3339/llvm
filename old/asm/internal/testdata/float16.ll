@a1 = global half 0xH3C00 ; 1.0

@a2 = global half 0xH4000 ; 2.0

@a3 = global half 0xHC000 ; -2.0

@a4 = global half 0xH7BFE ; 65472.0

@a5 = global half 0xH7BFF ; 65504.0

@a6 = global half 0xHFBFF ; -65504.0

@a7 = global half 0xH0000 ; 0.0

@a8 = global half 0xH8000 ; -0.0

@a9 = global half 0xH7C00 ; +inf

@a10 = global half 0xHFC00 ; -inf

@a11 = global half 0xH5B8F ; 241.875

@a12 = global half 0xH48C8 ; 9.5625
