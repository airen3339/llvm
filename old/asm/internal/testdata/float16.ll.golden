@a1 = global half 1.0

@a2 = global half 2.0

@a3 = global half -2.0

@a4 = global half 65472.0

@a5 = global half 65504.0

@a6 = global half -65504.0

@a7 = global half 0.0

@a8 = global half -0.0

@a9 = global half 0xH7C00

@a10 = global half 0xHFC00

@a11 = global half 241.875

@a12 = global half 9.5625
